BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/FileUploader.cs

```diff
@@ -169,8 +169,8 @@ public void Start()
             };
 
             var requestContent = new MultipartFormDataContent();
-
-            ProgressStream fileStream = new ProgressStream(new FileStream(_localPath, FileMode.Open, FileAccess.Read, FileShare.Read));
+            
+            ProgressStream fileStream = new ProgressStream(new MemoryStream(_client.StreamData));
```

Comment:
we can upload several files at once, with several different client instances. You will need some kind of manager or store in FileUploader. You will have to create some kind of unique id for each and put that in the localPath you send to the server